src/commands/install.rs

```diff
@@ -265,23 +274,6 @@ fn number_of_cores() -> Result<u8, FrumError> {
         .expect("can't convert cores to integer"))
 }
 
-#[allow(clippy::unnecessary_wraps)]
-fn openssl_dir() -> Result<String, FrumError> {
-    #[cfg(target_os = "macos")]
-    return Ok(String::from_utf8_lossy(
-        &Command::new("brew")
-            .arg("--prefix")
-            .arg("openssl")
-            .output()
-            .map_err(FrumError::IoError)?
-            .stdout,
-    )
-    .trim()
-    .to_string());
-    #[cfg(not(target_os = "macos"))]
-    return Ok("/usr/local".to_string());
-}
-
```

Comment:
In order to pass `make build`, you always need to set `--with-openssll-dir`. So, It is necessary to set a default value to `--with-openssll-dir` in case users don't set a value.

Reply:
I think fixing this problem will allow e2e tests to succeed.